type-definitions/tests/immutable-flow.js

```diff
@@ -173,6 +160,8 @@ stringToNumber = Map({'a': 'a'})
 stringToNumber = Map([['a', 1]])
 // $ExpectError
 stringToNumber = Map([['a', 'b']])
+// FIXME: this should trigger an error -- this is actually a Map<string, string>
+stringToNumber = Map(List.of(List(['a', 'b'])))
```

Comment:
Great find! Would you mind adding an issue for it? FIXMEs get lost pretty easily.